CONTRIBUTING.md

```diff
@@ -66,11 +66,11 @@ node . exec
 
 For example instead of:
 ```bash 
-npm install esbuild
+npm exec --yes false -- esbuild
```

Comment:
What's the thinking behind forcing `yes` to be false?

Reply:
Forcing ```yes``` to be false is not really necessary, as this is just an **example command**. We included it because it was a reference to the last issue we worked on.

Would you prefer if we made the command broader and changed it to: ```npm exec -- <package>``` and ```node . exec -- <package>``` ?

Reply:
Yeah suggestions, especially in readmes etc, should be as minimally prescriptive as possible.  Folks are going to be copying and pasting it, and anything we bake in that they may not need by default is potentially confusing to them.  Especially in this case the suggestion would fail for most folks who don't already have esbuild in their npx cache.

Reply:
So yes, let's make it broader, exactly like your updated suggestions.

Reply:
Just pushed those changes. Appreciate the explanation, that makes a lot of sense!